terracumber_config/tf_files/SUSEManager-Head-NUE.tf

```diff
@@ -179,7 +179,7 @@ module "cucumber_testsuite" {
     pool = "ssd"
     network_name = null
     bridge = "br0"
-    additional_network = "192.168.41.0/24"
+    additional_network = "192.168.99.0/24"
```

Comment:
As agreed with Eric, Uyuni uses `192.168.100.0/24`, so now head will use `192.168.99.0/24` so we don't need to change Head again.

After this is deployed, we'll need to taint head's extra network, or run a destroy.

Reply:
Thanks!